frontend/services/entities/mdm_apple.ts

```diff
@@ -26,13 +26,15 @@ export interface IAddVppAppPostBody {
   team_id: number;
   platform: ApplePlatform;
   self_service?: boolean;
+  automatic_install?: boolean;
   labels_include_any?: string[];
   labels_exclude_any?: string[];
 }
 
 export interface IEditVppAppPostBody {
   team_id: number;
   self_service?: boolean;
+  automatic_install?: boolean;
```

Comment:
Should skip this here, as well as the below, as we don't want to pass this on edit at this point. Runs the risk of, when we _do_ start allowing toggling automatic install in edit, overwriting titles to "don't auto-install" on unrelated edits, which we wouldn't want (though for now this is a no-op).